.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I prefer this configuration, just like in the extension. :)
```json 
{
  "singleQuote": true,
  "printWidth": 95,
  "tabWidth": 2
}
```

Reply:
np, changed